articles/100_ament.md

```diff
@@ -57,7 +57,7 @@ If the resulting artifacts are installed into `/usr`, for example, it might not
 
 [ament_tools](https://github.com/ament/ament_tools) is a Python package which provides the command line tool `ament` to build, test, install, and uninstall packages.
 It is similar to [catkin_tools](https://github.com/catkin/catkin_tools) and builds each package in a workspace in topological order.
-Support for different build systems is integrated through extension points which allows to contribute support for other build types without change the ament tool itself.
+Support for different build systems is integrated through extension points which allows to contribute support for other build types without changing the ament tool itself.
```

Comment:
as long as you're here, I think the words `to contribute` can be deleted as well

Reply:
Heh, I just changed that myself.  Not sure if you just want to remove "to contribute", or if you like my "allows the implementation of".  I'm fine either way.

Reply:
Reads fine to me!